File: appointment-management-service/src/main/java/com/healthcare/appointment/dtos/Patient.java
package com.healthcare.appointment.dtos;

import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class Patient{
    private Long id;
    private String firstName;
    private String lastName;
    private String phoneNumber;
    private String insuranceId;
    private String email;
    private Address address;
}

File: healthcare/src/app/patient-service/models/Patient.ts
export class Patient {
    id: number=0;
    firstName: string="";
    lastName: string="";
    phoneNumber: string="";
    insuranceId: string="";
    email: string="";
    address!: Address;
  }
  
  export class Address {
    id: number=0;
    street: string="";
    city: string="";
    state: string="";
    zipCode: string="";
  }
  